### ISSUE TITLE
Education benefits wizard doesn't load the form introduction properly when using Chrome with NVDA screen reader

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#6556]
## What happened?
When I'm navigating our education wizard button, I cannot load the first page of our education benefits form if I'm using the Chrome browser and NVDA screen reader. Screenshots and detailed information below.

## Specs:
- Device: Windows 10 laptop
- Browser: Chrome latest
- Screen reader: [NVDA](https://www.nvaccess.org/download/)

## Steps to Reproduce
- URL: https://staging.va.gov/education/how-to-apply/
- Tab to the `Find your education benefits` button and press SPACE or click to open
- Select the third tab for the Edith Nourse Rogers STEM scholarship
- Select YES from the second set of radio buttons
- Click or tab to the Apply now link and press ENTER
- Verify the new page is never announced by NVDA and you cannot navigate the page as expected
- If you load https://staging.va.gov/education/apply-for-education-benefits/application/1995/introduction in a new browser window, it works correctly

## Acceptance Criteria
- [ ] The screen reader should announce the new page and set focus on the breadcrumbs.

## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to Epic** (what body of work is this a part of? possibly `Ongoing Maintenance`)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `triage`, `tools-improvements`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with `Bug`**

## Screenshots
![Screen Shot 2020-03-04 at 4 46 19 PM](https://user-images.githubusercontent.com/934879/75930594-90476900-5e38-11ea-8bd4-7f560cf060ed.png)
